apps/financial-aid/web-veita/src/components/ApplicationFiltersProvider/ApplicationFiltersProvider.tsx

```diff
@@ -13,6 +13,7 @@ interface ApplicationFiltersProvider {
   setApplicationFilters?: React.Dispatch<
     React.SetStateAction<ApplicationFilters>
   >
+  loading?: boolean
```

Comment:
Are you sure you wan't to have this an optional boolean instead of having it a boolean with a default value?